py/py.mk

```diff
@@ -186,7 +186,6 @@ PY_CORE_O_BASENAME = $(addprefix py/,\
 	)
 
 PY_EXTMOD_O_BASENAME = \
-	extmod/moduasyncio.o \
```

Comment:
Remove the MicroPython C implementation of asyncio. I also recommend removing `moduselect`.